etc/kayobe/trivy/allowed-vulnerabilities.yml

```diff
@@ -0,0 +1,18 @@
+---
+###############################################################################
+# Trivy allowed vulnerabilities list
+
+# Example allowed vulnerabilities file setup
+#
+# keystone_allowed_vulnerabilities:
+#   - CVE-2022-2447
+#
+# barbican_api_allowed_vulnerabilities:
+#   - CVE-2023-31047
+
+global_allowed_vulnerabilities:
+  - CVE-2024-36039
```

Comment:
Do we need this one now that the requirements change has merged?

Reply:
Moved to the example file setup, so people know `global_allowed_vulnerabilities` variable can be used in the future.